styx-common/src/main/java/com/spotify/styx/storage/DatastoreStorage.java

```diff
@@ -548,8 +548,75 @@ void deleteResource(String id) throws IOException {
     return Optional.of(entityToBackfill(entity));
   }
 
-  List<Backfill> getBackfills() {
-    final EntityQuery query = Query.entityQueryBuilder().kind(KIND_BACKFILL).build();
+  List<Backfill> getBackfills(boolean showAll) {
```

Comment:
These methods could likely be refactored by defining a private method taking in `queryBuilder` and doing the actual query; the other methods just call it.